app/src/main/java/com/eventyay/organizer/core/attendee/list/AttendeesFragment.java

```diff
@@ -224,7 +231,20 @@ public boolean onQueryTextChange(String s) {
     }
 
     private void setupRecyclerView() {
-        fastItemAdapter = new ItemAdapter<>();
+
+        comparator = (o1, o2) -> {
+            if (sortingStrategy == SORTBYNAME) {
+                return o1.getFirstname().compareTo(o2.getFirstname());
+            } else if (sortingStrategy == SORTBYTICKET) {
+                return o1.getTicket().getType().compareTo(o2.getTicket().getType());
+            }
+            return 0;
+        };
+
+        ComparableItemListImpl<Attendee> itemListImpl = new ComparableItemListImpl<>(comparator);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'itemListImpl' should be declared final.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=4765425)